x-pack/plugins/apm/public/components/routing/app_root.tsx

```diff
@@ -5,32 +5,33 @@
  * 2.0.
  */
 
-import { ApmRoute } from '@elastic/apm-rum-react';
+// import { ApmRoute } from '@elastic/apm-rum-react';
```

Comment:
Should this be removed? Are we still using `ApmRoute` to record route-change transactions?

Reply:
Yeah, it should be removed, and I'll create a follow-up ticket to replace it with something custom. I don't think we can use ApmRoute anymore.